src/client/components/Search/useSearch.tsx

```diff
@@ -344,14 +395,17 @@ const mijnQuery = selector({
 
 export interface SearchResults {
   ma?: SearchEntry[];
+  ad?: Loadable<SearchEntry[] | null>;
```

Comment:
Zou jde resultaten kunnen toevoegen aan de Fuse instantie waar ook de MA resultaten in geladen worden? Ik denk dat dit beter is dan per "type" zoekresultaat onderscheid maken. Het onderscheid is nu:

- Mijn Amsterdam resultaten, iets waar wij zelf wat mee kunnen
- Amsterdam resultaten, alle overige van Amsterdam.nl

Je kunt misschien de top 10 resultaten van de BAG search inmixen. Mogelijk komt er niets naar voren bij een zoek actie omdat er andere, betere?, resultaten zijn (volgens Fuse).